src/DssSpell.t.sol

```diff
@@ -911,28 +955,282 @@ contract DssSpellTest is DssSpellTestBase {
+            assertEq(pool.usdsJoin(), address(usdsJoin), "pool: unexpected usdsJoin address");
+            assertEq(pool.daiJoin(),   address(daiJoin), "pool: unexpected daiJoin address");
+            assertEq(pool.stableDebt(),      stableDebt, "pool: unexpected stableDebt address");
+            assertEq(pool.variableDebt(),  variableDebt, "pool: unexpected variableDebt address");
+            assertEq(pool.redeemable(),  address(ausds), "pool: redeemable does not match");
+            assertEq(pool.king(),   address(pauseProxy), "pool: king does not match pause proxy");
+        }
+
+        // D3MOracle checks
+        {
+            assertEq(oracle.ilk(),                   ilk, "oracle: unexpected ilk");
+            assertEq(oracle.hub(),          address(hub), "oracle: unexpected hub address");
+            assertEq(oracle.vat(),          address(vat), "oracle: unexpected vat address");
+            assertEq(oracle.hub(),          address(hub), "oracle: unexpected vat address");
+        }
+
+         // D3MPlan checks
+        {
+            assertEq(plan.operator(), operator, "plan: unexpected operator address");
+            assertEq(plan.wards(address(mom)), 1,  "plan: mom not relied");
+            assertEq(plan.active(), true,  "plan: inactive");
+            // Ensure targetAssets can be updated by the operator
+            vm.prank(operator);
+            plan.setTargetAssets(100e18);
+            assertEq(plan.getTargetAssets(0), 100e18,  "plan: unexpected target assets value");
+        }
+
+         // Test D3MMom
+        {
+            assertEq(mom.authority(), address(chief), "mom: unexpected authority address");
+            // Deactivate the D3M via mom
+            vm.prank(DSChiefAbstract(chief).hat());
+            mom.disable(address(plan));
+            assertEq(plan.active(), false, "mom: still active");
+
+            hub.exec(ilk);
+            (uint256 ink, uint256 art) = vat.urns(ilk, address(pool));
+            assertLt(ink, WAD, "mom: unexpected ink amount"); // Less than some dust amount is fine (1 DAI)
+            assertLt(art, WAD, "mom: unexpected art amount");
+        }
+
+        // Test new ILK_REGISTRY
+        {
+            assertEq(reg.pos(ilk),    66, "ilk reg: unexpected postion");
+            assertEq(reg.join(ilk),   addr.addr("DIRECT_HUB"), "ilk reg: unexpected join address");
+            assertEq(reg.gem(ilk),    0x09AA30b182488f769a9824F15E6Ce58591Da4781, "ilk reg: unexpected gem address"); // ausds
+            assertEq(reg.dec(ilk),    18, "ilk reg: decimals"); // ausds has 18 decimals
+            assertEq(reg.class(ilk),  4, "ilk reg: unexpected class");
+            assertEq(reg.pip(ilk),    address(oracle), "ilk reg: unexpected pip address");
+            assertEq(reg.name(ilk),   "Aave Ethereum Lido USDS", "ilk reg: unexpected name"); // ausds name
+            assertEq(reg.symbol(ilk), "aEthLidoUSDS", "ilk reg: unexpected symbol"); // ausds symbol
+        }
```

Comment:
Can we also test [`xlip`](https://github.com/makerdao/ilk-registry/blob/1d65fb6e17c28e9e94ac88f0d8ccad04d8945f3c/src/IlkRegistry.sol#L114) is correctly set to `address(0)`?

Reply:
addded in https://github.com/makerdao/spells-mainnet/pull/432/commits/e8a3716e08d47d2a9149d8070b8dcfbcbaef0a58